.spelling

```diff
@@ -219,10 +219,6 @@ routes.go
 inlining
 Performant
 serializable
-OpenSearch
-Opensearch
```

Comment:
Removing these lines is causing linters to fail due to these terms existing in ADRs. These shouldn't be removed for that reason!